### PR TITLE
Initial creation

### PR DESCRIPTION
**Pull request by denis-yuen**
Initial work

### CODE REVIEW

**Review by garyluu — APPROVED** [review]
There's `-B` and `-ntp` that can be added

**Review by NatalieEO — APPROVED** [review]

File: .github/workflows/mvn.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: actions/checkout@v2
+      # Step that does that actual cache save and restore
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+           ${{ runner.os }}-maven-
+      - name: Build with mvn
+        run: mvn -B -ntp clean install -DskipTests
```
Comment (coverbeck) [inline .github/workflows/mvn.yml]: Why skipTests? Because there aren't any?

**Review by coverbeck — APPROVED** [review]
The idea is that you will manually, with HubFlow and maven release plugins, release this repo's artifacts into the OICR repo, right?

File: .github/workflows/mvn.yml (hunk 1 above)
Comment (denis-yuen) [reply .github/workflows/mvn.yml]: Think I carried this over from the main repo, but yeah it should do the same thing either way currently.

**Pull request merged**